scm/src/CMakeLists.txt

```diff
@@ -51,12 +51,19 @@ include(CMakeForceCompiler)
 
 SET(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake/modules)
 
-IF(DEFINED ENV{NETCDF})
-  MESSAGE(STATUS "The NETCDF environment variable is $ENV{NETCDF}")
-  SET(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} $ENV{NETCDF})
-ELSE(DEFINED ENV{NETCDF})
-  MESSAGE(FATAL_ERROR "The NETCDF environement variable must be set to point to your NetCDF installation before building. Stopping...")
-ENDIF(DEFINED ENV{NETCDF})
+#If using Hera, use the FindNetCDF.cmake module to find the NetCDF installed with NCEPLIBS; 
+#otherwise, rely on the NETCDF environment variable to point to the desired NetCDF installation
+if("$ENV{CMAKE_Platform}" STREQUAL "hera.intel")
+  MESSAGE(STATUS "Using FindNetCDF.cmake module for $ENV{CMAKE_Platform}")
+  FIND_PACKAGE(NetCDF REQUIRED COMPONENTS C CXX Fortran)
```

Comment:
We should clean this up at some point - all systems should be able to use the cmake FIND_PACKAGE functionality.

Reply:
Yes. Other systems are relying on users to set an environment variable at the moment.